### PR TITLE
Add more params + fix links

### PR DESCRIPTION
**Pull request by KaptenJansson**

### CODE REVIEW

File: src/web_app/html/params.html [hunk 1]
```diff
@@ -65,31 +65,33 @@
...
+        <tr><td><a href="https://appr.tc?stereo=true">stereo=true</a></td><td>Turn on stereo audio</td></tr>
+        <tr><td><a href="https://appr.tc?debug=loopback">debug=loopback</a></td><td>Connect to yourself, e.g. to test firewalls</td></tr>
+        <tr><td><a href="https://appr.tc?ts=https://turn.example.org">ts=[turnserver]</a></td><td>Set TURN server different from the default</td></tr>
+        <tr><td><a href="https://appr.tc?audio=true&amp;video=false">audio=true&amp;video=false</a></td><td>Audio only</td></tr>
+        <tr><td><a href="https://appr.tc?audio=false">audio=false</a></td><td>Video only</td></tr>
+        <tr><td><a href="https://appr.tc?audio=echoCancellation=false">echoCancellation=false</a></td><td>Disable all audio processing</td></tr>
+        <tr><td><a href="https://appr.tc?audio=googEchoCancellation=false">googEchoCancellation=false</a></td><td>Disable echo cancellation</td></tr>
+        <tr><td><a href="https://appr.tc?audio=googAutoGainControl=true">googAutoGainControl=true</a></td><td>Enable gain control</td></tr>
```
Comment (juberti) [inline src/web_app/html/params.html]: isn't AGC on by default? (and NR below)

File: src/web_app/html/params.html [hunk 2]
```diff
@@ -65,31 +65,33 @@
...
+        <tr><td><a href="https://appr.tc?audio=googNoiseReduction=true">audio=googNoiseReduction=true</a></td><td>Enable noise reduction</td></tr>
+        <tr><td><a href="https://appr.tc?asc=ISAC/16000">asc=ISAC/16000</a></td><td>Set preferred audio send codec to be ISAC at 16kHz (use on Android)</td></tr>
+        <tr><td><a href="https://appr.tc?arc=opus/48000">arc=opus/48000</a></td><td>Set preferred audio receive codec Opus at 48kHz</td></tr>
+        <tr><td><a href="https://appr.tc?dtls=false">dtls=false</a></td><td>Disable <a href="https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security" title="Wikipedia article: DTLS">Datagram Transport Layer Security</a></td></tr>
```
Comment (juberti) [inline src/web_app/html/params.html]: we probably should stop advertising this one

File: src/web_app/html/params.html (hunk 1 above)
Comment (KaptenJansson) [inline src/web_app/html/params.html]: True, fixed.

File: src/web_app/html/params.html (hunk 2 above)
Comment (KaptenJansson) [inline src/web_app/html/params.html]: Removed.

**Comment by juberti** [comment]
lgtm

**Pull request merged**